galsim/roman/roman_psfs.py

```diff
@@ -49,22 +56,29 @@ def getPSF(SCA, bandpass,
     (Note: the files at that url still use the old WFIRST name.  We have renamed them to use the
     new name of the telescope, Roman, after downloading.)
 
-    The default is to do the calculations using the full specification of the Roman pupil plane,
-    which is a costly calculation in terms of memory.  For this, we use the provided pupil plane for
-    long- and short-wavelength bands for Cycle 7 (the list of bands associated with each pupil plane
-    is stored in ``galsim.roman.longwave_bands`` and ``galsim.roman.shortwave_bands``).
-
-    To avoid using the full pupil plane configuration, use the optional keyword
-    ``approximate_struts``.  In this case, the pupil plane will have the correct obscuration and
-    number of struts, but the struts will be purely radial and evenly spaced instead of the true
-    configuration.  The simplicity of this arrangement leads to a much faster calculation, and
-    somewhat simplifies the configuration of the diffraction spikes.  Also note that currently the
-    orientation of the struts is fixed, rather than rotating depending on the orientation of the
-    focal plane.  Rotation of the PSF can easily be affected by the user via::
+    The mask images for the Roman pupil plane are available at from the Roman Reference Information
+    page: https://roman.gsfc.nasa.gov/science/Roman_Reference_Information.html.
+    There are separate files for each SCA, since the view of the spider pattern varies somwhat
+    across the field of view of the wide field camera. Furthermore, the effect of the obscuration
+    is somewhat different at longer wavelengths, so F184 has a different set fo files than the
+    other filters.  cf. the ``galsm.roman.longwave_bands`` and ``galsim.roman.shortwave_bands``
+    attributes, which define which bands use which pupil plane images.
+
+    To avoid using the full pupil plane configuration, use the optional keyword ``pupil_bin``.
+    The full pupil-plane images are 4096 x 4096, which is more detail than is typically needed for
+    most applications. The default binning is 4x4, which results in an image that is 1024 x 1024.
+    This provides enough detail for most purposes and is much faster to render than using the full
+    pupil plane image.  Using pupil_bin=8 (resulting in a 512 x 512 image) still provides fairly
+    reasonable results and is even faster to render, but it is not recommended to use higher
+    binning than that, as the diffraction spikes will be noticeably degraded.
+
+    Also note that currently the orientation of the struts is fixed, rather than rotating depending
+    on the orientation of the focal plane.  Rotation of the PSF can easily be affected by the user
+    via::
```

Comment:
```suggestion
    via:
```

Reply:
This apparent typo is actually Sphinx markup.  The double colon, line skip, and indent is the start of a code block.

Reply:
Ah ha, thanks for the reminder!